aerospike-mixin/dashboards/aerospike-overview.libsonnet

```diff
@@ -0,0 +1,1209 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'aerospike-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local nodesPanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(job, aerospike_cluster, instance) (aerospike_namespace_ns_cluster_size{' + matcher + '})',
```

Comment:
I would get rid of the job column from all the tables in this dashboard. This most of the times will have a single value, so exlcuing it will open up space for more relevant information.